cypress/e2e/augury.cy.ts

```diff
@@ -57,4 +57,34 @@ describe("Augury", () => {
   it("creates flights and calculates inventory?", () => {
     // TODO
   });
+
+  it("creates and deletes creatives", () => {
+    const now = new Date().toISOString();
+    const canary = `Acceptance Test: ${now}`;
+    const audioFile = "cypress/samples/two-tone.mp3";
+
+    cy.visit("/creatives");
+    cy.get("h1").contains("Creatives");
+
+    // create creative
+    cy.get("a.btn-success[href='/creatives/new']").click();
+    cy.contains("New Creative");
```

Comment:
If this is intended to test the loading of the **New Creative** form page, I think it probably currently isn't necessarily doing that. The button that was clicked on the previous line also contains the text `New Creative`, so this would succeed, even if the navigation fails. (When I run this test, what I'm seeing in step 6 of the log is Cypress matching `-contains` on the button, not on the header of the form page).